apache-rat-core/src/main/java/org/apache/rat/ReportConfiguration.java

```diff
@@ -179,31 +177,31 @@ public boolean isDryRun() {
     /**
      * @return The filename filter for the potential input files.
      */
-    public FilenameFilter getInputFileFilter() {
-        return inputFileFilter;
+    public FilenameFilter getFilesToIgnore() {
+        return filesToIgnore;
     }
 
     /**
-     * @param inputFileFilter the filename filter to filter the input files.
+     * @param filesToIgnore the filename filter to filter the input files.
      */
-    public void setInputFileFilter(FilenameFilter inputFileFilter) {
-        this.inputFileFilter = inputFileFilter;
+    public void setFilesToIgnore(FilenameFilter filesToIgnore) {
+        this.filesToIgnore = filesToIgnore;
     }
 
-    public IOFileFilter getDirectoryFilter() {
-        return directoryFilter;
+    public IOFileFilter getDirectoriesToIgnore() {
+        return directoriesToIgnore;
     }
 
-    public void setDirectoryFilter(IOFileFilter directoryFilter) {
-        if (directoryFilter == null) {
-            this.directoryFilter = FalseFileFilter.FALSE;
+    public void setDirectoriesToIgnore(IOFileFilter directoriesToIgnore) {
```

Comment:
ScanDefault could have boolean hasFiles()/hasDirectories() methods to not have to handle null values while interacting with the scan configuration